### DISCUSSION TITLE
How to return to separate views for each method in a single livewire controller

### DISCUSSION THREAD

**Discussion by toxifiedmatrix** [livewire/livewire#2071] [1. Help]
In a normal controller, we can return to a view with initialized variables passing to the view, without any limit we can return to different blade files as views and can possibly return to separate views from each and every method, from the controller. In livewire I can only return to a single view which is under render function. How can I do the same in the Livewire Controller.

**Comment by joshhanley** [comment]
Livewire doesn't work that way as they aren't controllers, they are components. As such they are more like Laravel Blade Components which each have a single render method. 

If you need multiple methods with views, just use a regular Laravel controller which returns views with Livewire Components in them.

Hope this helps!

**Reply by toxifiedmatrix** [reply]
Yes, It does helps, Ill surely follow that thanks for the knowledge

**Reply by joshhanley** [reply]
No worries!
